lib/newgistics/update_shipment.rb

```diff
@@ -0,0 +1,27 @@
+module Newgistics
+  class UpdateShipment
```

Comment:
I was thinking we could name this as a noun and not a verb to keep it inline with the other models we have, maybe we could name it `Newgistics::ShipmentUpdate`? What do you think?